UPGRADING.md

```diff
@@ -62,12 +62,6 @@ import (
 )
 ```
 
-## [v2.0.x](https://github.com/cosmos/interchain-security/releases/tag/v2.0.0)
+## [v4.0.x](https://github.com/cosmos/interchain-security/tree/release%2Fv4.0.x)
```

Comment:
Is this the right link? Just wondering since the old link was to a release, this is just to the repo root for the commit.